android_mic_streaming/app/src/main/java/org/mozilla/deepspeechdemo/MainActivity.kt

```diff
@@ -0,0 +1,137 @@
+package org.mozilla.deepspeechdemo
+
+import android.Manifest
+import android.content.pm.PackageManager
+import android.media.AudioFormat
+import android.media.AudioRecord
+import android.media.MediaRecorder
+import android.os.Build
+import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.app.ActivityCompat
+import kotlinx.android.synthetic.main.activity_main.*
+import org.mozilla.deepspeech.libdeepspeech.DeepSpeechModel
+import org.mozilla.deepspeech.libdeepspeech.DeepSpeechStreamingState
+
+
+class MainActivity : AppCompatActivity() {
+    private var model: DeepSpeechModel? = null
+    private var streamContext: DeepSpeechStreamingState? = null
+
+    private val BEAM_WIDTH = 50
+    private val LM_ALPHA = 0.75f
+    private val LM_BETA = 1.85f
```

Comment:
People will need to change that value depending on their model's.

Reply:
I'll add a comment. And make a hint in the readme.